.devcontainer/Dockerfile

```diff
@@ -310,14 +312,45 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     make -j"$(nproc)" && \
     make install && \
     cd / && \
```

Comment:
Why change the folder?
This can be refactored, if absolute path are used in deletion. This might even be considered safer.